ethr-status-registry/src/main/java/me.uport.sdk.ethrstatusregistry/EthrStatusRegistry.kt

```diff
@@ -0,0 +1,88 @@
+package me.uport.sdk.ethrstatusregistry
+
+import kotlinx.coroutines.runBlocking
+import me.uport.credential_status.CredentialStatus
+import me.uport.credential_status.StatusEntry
+import me.uport.credential_status.StatusResolver
+import me.uport.sdk.core.Networks
+import me.uport.sdk.jsonrpc.JsonRPC
+import me.uport.sdk.jwt.JWTTools
+import org.kethereum.extensions.hexToBigInteger
+import org.kethereum.keccakshortcut.keccak
+import pm.gnosis.model.Solidity
+
+class EthrStatusRegistry : StatusResolver {
+
+    override val method = "EthrStatusRegistry2019"
+
+    override fun checkStatus(credential: String): CredentialStatus {
```

Comment:
this should be a `suspend` method. Thi would make `runBlocking` unnecessary here, which is good since it should not be used in this method